### ISSUE TITLE
Allow playing Youtube channel's live stream

### ISSUE THREAD

**Issue by ispal** [sampotts/plyr#789]
This is more of a feature request. It would be nice to allow Youtube plugin to support playing channel's live stream. Live stream is currently in the following URL

https://www.youtube.com/embed/live_stream?channel=channel-id

I think it currently uses Youtube IFrame API and passes video id to it, but that doesn't work with the live streams. It would be nice to check if the URL includes live stream and if it does, don't fetch video using the video id, just ignore it. In that case, Youtube will use the **src** attribute from the iframe. Of course there's need to option to change url dynamically as well.

- [x ] Issue does not already exist
- [ ] Issue observed on https://plyr.io

### Expected behaviour 
Allow playing Youtube luve stream

### Actual behaviour
It doesn't exist?

### Environment

- Browser:
- Version:  
- Operating System:
- Version: 

Players affected:
- [x] HTML5 Video
- [ ] HTML5 Audio
- [x] YouTube
- [ ] Vimeo

### Steps to reproduce 
- 

### Relevant links
Example in the docs: https://developers.google.com/youtube/iframe_api_reference#Examples

**Comment by sampotts** [comment]
Good idea 👍

**Comment by gurupras** [comment]
This already seems to be working. Here's a [sample](https://jsfiddle.net/gurupras/qrsqnato/)

**Comment by ispal** [comment]
Yeah, but you need to know the video ID. What I suggest is using channel's live stream URL so it doesn't matter what the video ID is.

**Comment by ispal** [comment]
Any plans to implement this in near future?

**Comment by friday** [comment]
If you need it you are welcome to PR this.

* https://github.com/sampotts/plyr/wiki/FAQ#4-whats-the-status-on--
* https://github.com/sampotts/plyr/blob/master/CONTRIBUTING.md#requesting-features-and-improvements
